components/shared/Sidebar.tsx

```diff
@@ -1,3 +1,5 @@
+'use client'
```

Comment:
You may remove this directive. We're using the pages router, so all are client components.